qiskit/circuit/library/generalized_gates/rv.py

```diff
@@ -55,10 +55,10 @@ def __init__(self, v_x, v_y, v_z, basis="U"):
             v_y (float): y-component
             v_z (float): z-component
             basis (str, optional): basis (see
-                :class:`~qiskit.quantum_info.synthesis.one_qubit_decompose.OneQubitEulerDecomposer`)
+                :class:`~qiskit.synthesis.one_qubit.one_qubit_decompose.OneQubitEulerDecomposer`)
         """
         # pylint: disable=cyclic-import
```

Comment:
Does pylint still detect a cyclic import now that the import is coming from `qiskit.synthesis` instead of `qiskit.quantum_info`?

Reply:
it seems that there are cyclic import lint errors if this disable is removed.